### ISSUE TITLE
Show additional information in progress bar? 

### ISSUE THREAD

**Issue by devmotion** [julia-vscode/julia-vscode#3317]
Is it possible to show additional information (messages, variables, etc.) in or combined with the progress bars in VSCode?

The more general issue for ProgressLogging is https://github.com/JuliaLogging/ProgressLogging.jl/issues/23, but I was wondering if there exists a way to show such logging information in VSCode specifically. As mentioned in the ProgressLogging issue (https://github.com/JuliaLogging/ProgressLogging.jl/issues/23#issuecomment-594307284), in Juno it was possible to display an additional message as a tooltip. I tried the example for Juno but it seems VSCode does not support the `message` key. Generally, I'm not sure either if a tooltip would be the best way to present more information since opening the progressbar popup in VSCode can be a bit inconvenient IMO.

**Comment by pfitzseb** [comment]
Yup, we currently only support the progress bar name. Can't quite remember why that is.
